src/sgame/sg_utils.cpp

```diff
@@ -781,11 +781,11 @@ void G_TeamToClientmask( team_t team, int *loMask, int *hiMask )
 
 	*loMask = *hiMask = 0;
 
-	for ( clientNum = 0; clientNum < MAX_CLIENTS; clientNum++ )
+	for ( clientNum = 0; clientNum < level.maxclients; clientNum++ )
 	{
-		client = g_entities[ clientNum ].client;
```

Comment:
This looks wrong. You have to check whether the client entity is in use.

Reply:
I guess it could be checked, but on the other hand I don't think this is a problem: this is a mark used by daemon to know if it should send entity infos to client, if a client doesn't exist, it doesn't matter if it is allowed or not to access an entity

Reply:
Still the function ought to work as advertised and not return wrong results.

Reply:
I've added a commit for that too